api/v1alpha1/condition_consts.go

```diff
@@ -9,15 +9,33 @@ const (
 	ReadyCondition ConditionType = "Ready"
 )
 
+const (
+	// VirtualMachinePreReqReadyCondition is set on VirtualMachines when all of a VirtualMachine's prerequisites are satisfied.
+	// For now, this check only validates that the VirtualMachineClass specified in the VirtualMachineSpec exists on the API
+	// server and the VirtualMachine's namespace has access to the specified VirtualMachineClass.
+	VirtualMachinePreReqReadyCondition = "VirtualMachinePreReqReady"
+
+	// VirtualMachinePreReqReadyReason documents that all of a VirtualMachine's PreRequisites have been satisfied.
```

Comment:
Since the condition type already identifies that this is for VirtualMachinePreReqReady, seems redundant to also have the Reason be that verbose, it can just be "Ready" like other conditions.